syntax = "proto3";

package events;

service EventService {
    rpc ReportEvent(EventRequest) returns (EventResponse);
}

message EventRequest {
    string event_type = 1;
    string event_data = 2;
    string event_id = 3;
}

message EventResponse {
    string status = 1;
}
